.github/scripts/extract-cache-keys.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/config.sh
+
+job_keys_json=$1
+
+toolchain_cache_key=$(echo $job_keys_json | jq -r ".[\"$TOOLCHAIN_NAME-toolchain-cache-key\"]")
```

Comment:
If dynamic matrix job outputs would be possible this script won't be needed, right?

Reply:
It looks like the script is needed.